packages/@react-native-windows/cli/src/config/dependencyConfig.ts

```diff
@@ -264,49 +202,119 @@ export function dependencyConfigWindows(
     for (const foundProject of foundProjects) {
       const projectFile = path.join(sourceDir, foundProject);
 
-      const projectLang = configUtils.getProjectLanguage(projectFile);
+      result.projects.push(calculateProjectFromFile(projectFile, sourceDir));
+    }
+  }
+
+  return result;
+}
 
-      const projectContents = configUtils.readProjectFile(projectFile);
+function calculateProjectFromFile(projectFile: string, sourceDir: string) {
```

Comment:
Nit: can You rename this, or comment that is only meant to work with projects that have already passed the check that they are RNW direct dependency projects (i.e. they include the props/targets)? This function assumes that, and wouldn't stop if you passed in any random vcxproj or csproj.